api/v1alpha1/discoveredcluster_types.go

```diff
@@ -33,6 +33,7 @@ type DiscoveredClusterSpec struct {
 	APIURL            string       `json:"apiUrl" yaml:"apiUrl"`
 	CreationTimestamp *metav1.Time `json:"creationTimestamp,omitempty" yaml:"creationTimestamp,omitempty"`
 	ActivityTimestamp *metav1.Time `json:"activityTimestamp,omitempty" yaml:"activityTimestamp,omitempty"`
+	Type              string       `json:"type" yaml:"type"`
```

Comment:
@JakobGray can you confirm this is now mandatory (i.e. that i fixed it)

Reply:
It doesn't have to be a required field (many of them aren't), but if you want it to be you need to regenerate the CRD/CSV (`make bundle` should do it). You'll see it added to the required list in the CRD.

Reply:
will do